testing/percona-xtradb-cluster/fix-posix_timers.patch

```diff
@@ -0,0 +1,50 @@
+--- a/mysys/posix_timers.c
++++ b/mysys/posix_timers.c
+@@ -188,10 +188,20 @@
+           On error, -1 is returned, and errno is set to indicate the error.
+ */
+
++struct linux_sigevent {
```

Comment:
This is horribly wrong hack. See PR #808 review comments on how to fix this properly.